test/Elastic.Apm.AspNetCore.Tests/DistributedTracingAspNetCoreTests.cs

```diff
@@ -191,8 +192,25 @@ public async Task DisposeAsync()
 			_agent2?.Dispose();
 		}
 
-		private async Task ExecuteAndCheckDistributedCall()
+		/// <summary>
+		/// Executes the distributed tracing call between the 2 services.
+		/// </summary>
+		/// <param name="PreventStartingActivity">If true, we'll create an actvity on .NET 5</param>
+		/// <returns></returns>
+		private async Task ExecuteAndCheckDistributedCall(bool PreventStartingActivity = false)
```

Comment:
Would calling `PreventStartingActivity` something like `startActivityBeforeHttpCall` (and have boolean semantics that are the inverse of current impl) be a better description for what the parameter controls?

Reply:
Sounds ok to me - I changed it. In all tests we create the activity, except in 1, that's why I called it `PreventStartingActivity`. But this can change later... I renamed it.